spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :request do
```

Comment:
- [ ] Your test is failed. I can't provide screenshots cause my machine is crashing.    But please make sure to make fix all the errors in your test and make them passed 👍

Reply:
@codecaiine ,

Thanks a lot for the review. Below, is a screenshot of the tests on my machine. I don't understand where the problem comes from but all the tests are passing here.

![screenshot controllers specs](https://github.com/Uthmanbello/myblog/assets/112436398/c37c72c8-23bb-4d5c-bab6-973c87f3b62c)

Please, I need clarification before requesting for another review.
Thanks as always.

Reply:
@Uthmanbello Good job. Maybe the errors came from my machine. Please you can submit it again another reviewer will check it 👍